truss/truss_config.py

```diff
@@ -527,6 +529,9 @@ def from_dict(d):
             input_type=d.get("input_type", DEFAULT_MODEL_INPUT_TYPE),
             model_metadata=d.get("model_metadata", {}),
             requirements_file=d.get("requirements_file", None),
+            _requirements_file_requirements=d.get(
```

Comment:
Just to be clear, there isn't a chance that this winds up in the yaml somehow, right?

Reply:
in the config yaml? We don't write to their config yaml AFAICT. Is there someone that I should check with?